pkgs/state-proving-manager/src/manager.zig

```diff
@@ -1,15 +1,48 @@
-const types = @import("types")
-const state_transition = @import("state-transition")
+const std = @import("std");
+const ssz = @import("ssz");
+const types = @import("@zeam/types");
+const state_transition = @import("@zeam/state-transition");
+const allocator = std.heap.page_allocator;
 
+pub const ZKVMContext = struct {
+    program_path: []const u8,
+    output_dir: []const u8,
+    backend: ?[]const u8 = null,
+};
 
-const StateTransitionOpts = struct{
-    zk_vm: ZKVM_TYPES,
-}
+extern fn powdr_prove(serialized: [*]const u8, len: usize, output: [*]u8, output_len: usize, binary_path: [*]const u8, binary_path_length: usize, result_path: [*]const u8, result_path_len: usize) void;
+
+pub const zkvm_configs: []const ZKVMContext = &.{
```

Comment:
we can probably transition it to a enum `union` struct

Reply:
makes sense, but since there's only one right now, let's leave it as is and change it in a future PR when we support proving more than one zkvm